src/genophenocorr/model/_variant.py

```diff
@@ -377,13 +379,13 @@ def create_variant_from_scratch(variant_coordinates: VariantCoordinates,
                                     is_preferred: bool,
                                     consequences: typing.Iterable[VariantEffect],
                                     exons_effected: typing.Sequence[int],
-                                    protein: typing.Sequence[ProteinMetadata],
+                                    protein_id: typing.Sequence[str],
```

Comment:
This should probably be `typing.Optional[str]` instead of `typing.Sequence[str]`, since an optional `str` is what `TranscriptAnnotation` expects.